components/src/CartLineItem/cart.lineitem.less

```diff
@@ -34,108 +37,111 @@
   }
 
   .thumbnail-col {
+    grid-row: 1 / last-line;
     position: relative;
     display: inline-block;
-    margin-right: 10px;
     vertical-align: top;
     img {
-      max-width: 60px;
+      border: 1px solid #D4D3D3;
```

Comment:
We have this declared in common.less as `@firstBorderColor`, so let's be sure to use that to allow for easier theming.